src/external/rawspeed/data/cameras.xml

```diff
@@ -5734,6 +5734,40 @@
 		<Crop x="0" y="0" width="-74" height="0"/>
 		<Sensor black="15" white="3971"/>
 	</Camera>
+	<Camera make="Panasonic" model="DMC-LX15">
+		<ID make="Panasonic" model="DMC-LX15">Panasonic DMC-LX15</ID>
+		<CFA width="2" height="2">
+			<Color x="0" y="0">GREEN</Color>
+			<Color x="1" y="0">BLUE</Color>
+			<Color x="0" y="1">RED</Color>
+			<Color x="1" y="1">GREEN</Color>
+		</CFA>
+		<Crop x="0" y="0" width="0" height="0"/>
+		<Sensor black="142" white="4095" iso_list="80 100 125 400"/>
```

Comment:
Ok, whitelevels are my error, but:

with ISO1600 samples i get different results depending on the tool i use:
dngmeta.sh panasonic_lumix_dmc_lx15_68.dng: Sensor black="**146**" white="4095"/

dngmeta.rb panasonic_lumix_dmc_lx15_68.dng: 
```
sensors [[1600, [145, 4095]]]
mostfrequent [[145, 4095], [1600]]
invsensors {}

        <Camera make="Panasonic" model="DMC-LX15">
                <ID make="" model="">Panasonic DMC-LX9</ID>
                <Sensor black="145" white="4095"/>
        </Camera>
```